draft-ietf-ppm-dap.md

```diff
@@ -1546,33 +1606,100 @@ will occur before aggregation.
 
 #### Example
 
+Successful upload
+
 ~~~ http
 POST /leader/tasks/8BY0RzZMzxvA46_8ymhzycOB9krN-QIGYvg_RsByGec/reports
 Host: example.com
-Content-Type: application/dap-report
+Content-Type: application/dap-upload-req
```

Comment:
The examples should include `Content-Length` since we're going to have a MUST about the requests and responses including them.

Reply:
@tgeoghegan is the answer "no" here given the discussion on https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/686#discussion_r2108267711?

Reply:
Is the answer "no" to what?

Reply:
sorry, I mean this example shouldn't include the Content-Length header.